lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -5001,26 +5002,14 @@ void SharedEmitterState::emitOps(EmissionList &thingsToEmit, raw_ostream &os,
   }
 }
 
-/// Prepare the given MLIR module for emission.
-static void prepareForEmission(ModuleOp module,
-                               const LoweringOptions &options) {
-  SmallVector<HWModuleOp> modulesToPrepare;
-  module.walk([&](HWModuleOp op) { modulesToPrepare.push_back(op); });
-  parallelForEach(module->getContext(), modulesToPrepare,
-                  [&](auto op) { prepareHWModule(op, options); });
-}
-
 //===----------------------------------------------------------------------===//
 // Unified Emitter
 //===----------------------------------------------------------------------===//
 
-LogicalResult circt::exportVerilog(ModuleOp module, llvm::raw_ostream &os) {
-  // Prepare the ops in the module for emission and legalize the names that will
-  // end up in the output.
-  LoweringOptions options(module);
-  prepareForEmission(module, options);
+static LogicalResult exportVerilogImpl(ModuleOp module, llvm::raw_ostream &os) {
```

Comment:
So these "Impl's":
1) Assume/Require PrepareForEmission has run
2) Assume/Expect LoweringOptions that should be used have been attached to the module already.

Sounds good to me, just checking their usage/expectations.

Reply:
They will use the default if there are no lowering options.  But yes.  Also will be broken out a bit better soon.